site/content/events/2016-ohio/_email_organizers.txt

```diff
@@ -0,0 +1 @@
+organizers-ohio-2015@devopsdays.org
```

Comment:
This needs to be 2016 (which you need to email info@devopsdays.org and ask them to set it up. You need to say who's in/out from last year and/or give the list of emails to use.)